utils/catatonit/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=catatonit
+PKG_VERSION:=0.1.7
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/openSUSE/catatonit.git
+PKG_SOURCE_DATE:=2022-03-07
+PKG_SOURCE_VERSION:=d8d72fea155c144ed3bf298a35a1aba5625a5656
+PKG_MIRROR_HASH:=skip
```

Comment:
This does not look good.

Reply:
Sorry, I always compile it with tests here to get hash, because hash that I compile locally usually does not match with hash here. Just had to do some work between, so I did not have time to fix this until now.. Anyway, test build has finished and was cancelled due to this and reported correct hash and it's now in it's place.